flecsi/util/hashtable.hh

```diff
@@ -31,6 +31,7 @@ struct hashtable;
 template<class KEY, class TYPE, class HASH>
 class hashtableIterator
 {
+  using key_t = KEY;
```

Comment:
This doesn't seem to be used anywhere.

Reply:
It's being usedhere: https://github.com/flecsi/flecsi/blob/cbe04e617dafb5da8bd3022cd100f9b84ecc5b7c/flecsi/util/hashtable.hh#L53

Reply:
Weird; it must have been refactored in a way that the unit tests aren't catching.